src/ergw_aaa_diameter_srv.erl

```diff
@@ -146,9 +146,11 @@ await_response(Promise, Timeout) ->
 
 handle_plain_error(Module, Error, Request, SvcName, CallOpts)
   when is_atom(Module) ->
+    ?LOG(error, "DIAMETER error in ~p of request type ~p with: ~p", [Module, hd(Request), Error]),
```

Comment:
`hd(Request)` is not safe.

Request is of type `message() = record() | maybe_improper_list()`, see http://erlang.org/doc/man/diameter_codec.html#message

Reply:
Fixed. I suppose info about request can be helpful in debug, I keep it in log